components/greeting.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react'
+
+export default () => {
+  const [slug, setSlug] = useState('')
+
+  return (
+    <div className="page">
+      <div className="title">Hello, </div>
+      <div className="subtitle">To whom am I speaking? </div>
+      <input type="text" name="search" onChange={(event) => { setSlug(event.target.value) }} value={slug} />
+      <div className="slug" />
+      <button type="button" onClick={() => { setSlug('') }}>Clear</button>
+      <div className="response">
+        Nice to meet you
+        {' '}
+        {slug}
+      </div>
```

Comment:
I would make this conditional so if name is empty you don't show anything (assumes you call the variable `name`)

```JavaScript
<div className="response">{ name ? `Nice to meet you ${name}` : null }</div>
```